### PR TITLE
Removed duplicate WC actions

### PR DESCRIPTION
**Pull request by Misplon**
Removed duplicated WooCommerce actions.

### CODE REVIEW

File: woocommerce/template-tags.php [hunk 1]
```diff
@@ -17,17 +17,10 @@ function siteorigin_corp_woocommerce_change_hooks() {
...
 	remove_action( 'woocommerce_after_shop_loop_item_title', 'woocommerce_template_loop_rating', 5 );
-	add_action( 'woocommerce_before_shop_loop_item_title', 'siteorigin_corp_woocommerce_archive_product_image', 10 );
-	remove_action( 'woocommerce_before_shop_loop_item', 'woocommerce_template_loop_product_link_open', 10 );
-	remove_action( 'woocommerce_after_shop_loop_item', 'woocommerce_template_loop_product_link_close', 5 );
-	remove_action( 'woocommerce_after_shop_loop_item', 'woocommerce_template_loop_add_to_cart', 10 );
-	remove_action( 'woocommerce_before_shop_loop_item_title', 'woocommerce_template_loop_product_thumbnail', 10 );
 	add_action( 'woocommerce_shop_loop_item_title', 'woocommerce_template_loop_product_link_open', 5 );
 	add_action( 'woocommerce_shop_loop_item_title', 'woocommerce_template_loop_product_link_close', 15 );
-	remove_action( 'woocommerce_after_shop_loop_item_title', 'woocommerce_template_loop_rating', 5 );
+	add_action( 'woocommerce_before_shop_loop_item_title', 'siteorigin_corp_woocommerce_archive_product_image', 10 );
```
Comment (AlexGStapleton) [inline woocommerce/template-tags.php]: Is there a specific reason why you're setting a priority? The default priority is `10` so specifying it is unnecessary.

Comment (Misplon) [reply woocommerce/template-tags.php]: That line appears as new but isn't. I just moved the order but didn't change the priority. We can look at this but ideally in another PR or in develop.

Comment (Misplon) [reply woocommerce/template-tags.php]: Removed_duplicate_WC_actions_by_Misplon_·_Pull_Request__89_·_siteorigin_siteorigin-corp

Comment (AlexGStapleton) [reply woocommerce/template-tags.php]: Ah. Didn't notice that.

**Pull request merged**